tests/quantization/test_saturation_issue_export.py

```diff
@@ -138,14 +141,81 @@ def test_without_saturation_helper(target_device):
         test_without_saturation_helper(device)
 
 
-def test_are_onnx_exported_weights_tensors_clipped(tmp_path):
+class EightConvTestModel(nn.Module):
+    def __init__(self, in_out_ch):
+        super().__init__()
+        self.features = []
+        self.features.append(nn.Sequential(create_conv(*in_out_ch[0], 2, -1, -2)))
+        self.features.append(nn.Sequential(create_conv(*in_out_ch[1], 5, 1, 1)))
+        self.features.append(nn.Sequential(create_conv(*in_out_ch[2], 1, 2, 2)))
+        self.features.append(nn.Sequential(create_conv(*in_out_ch[3], 9, -1, 0)))
+        self.features.append(nn.Sequential(create_conv(*reversed(in_out_ch[3]), 3, 0, 1)))
+        self.features.append(nn.Sequential(create_conv(*reversed(in_out_ch[2]), 1, -1, 9)))
+        self.features.append(nn.Sequential(create_conv(*reversed(in_out_ch[1]), 2, 10, 1)))
+        self.features.append(nn.Sequential(create_conv(*reversed(in_out_ch[0]), 1, 1, 1)))
+        self.features = nn.Sequential(*self.features)
+
+    def forward(self, x):
+        return self.features(x)
+
+
+def test_are_fq_exported_per_channel_weights_tensors_clipped(tmp_path):
+    in_out_ch = [[1, 3], [3, 5], [5, 7], [7, 10]]
+    model = EightConvTestModel(in_out_ch)
+    nncf_config = get_config_for_export_mode(False)
+    nncf_config.update({"input_info": {
+        "sample_size": [1, 1, 20, 20]
+    }})
+
+    _, compression_ctrl = create_compressed_model_and_algo_for_test(model, nncf_config)
+
+    # Update scale tensors in the model
+    quantizers = compression_ctrl.weight_quantizers.values()
+    for quantizer in quantizers:
+        quantizer.quantizer_module_ref.scale = torch.nn.Parameter(
+            torch.rand(quantizer.quantized_module.weight.shape[0], dtype=torch.float32, requires_grad=True))
```

Comment:
`torch.rand_like`

Reply:
I need only first dimension which is filter number, so I use `torch.rand`